packages/esm-patient-chart-app/src/visit-header/confirmation-modal.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { Button, ModalBody, ModalFooter, ModalHeader } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+
+interface ConfirmationDialogProps {
+  closeDialog: () => void;
+  leavePatientChart: (e: any) => void;
+  props: any;
+}
+
+const ConfirmLeavingPatientChart: React.FC<ConfirmationDialogProps> = ({ closeDialog, leavePatientChart, props }) => {
+  const { t } = useTranslation();
+  const { type = '', title = '', additionalProps = {} } = props;
+  const workspaceTitle = additionalProps && additionalProps?.workspaceTitle;
+
+  const unifiedTitle = title.includes('Form') ? title : `${title} Form`;
```

Comment:
This needs to be localizable if we're using it. It's not clear to me why we need this though?

Reply:
Just for uniformity because some form titles don't have the word form and the look off and inconsistent.